HISTORY.md

```diff
@@ -22,8 +22,9 @@ rn = @reaction_network begin
     @parameters η
     k, 2X --> X2, [noise_scaling=η]
 end
-getnoisescaling(rn)
+get_noise_scaling(rn)
```

Comment:
This and other related changes is due to this one building on the noise scaling PR